chart/epinio/templates/stage-scripts.yaml

```diff
@@ -69,7 +70,7 @@ data:
     # is placed into a trap to ensure that it will always run, even for a staging failure.
     #
     set -e
-    trap "curl -X POST http://localhost:4191/shutdown || true" EXIT
+    trap "curl -X POST http://localhost:4191/shutdown 2> /dev/null || true" EXIT
```

Comment:
What's the motivation to hid the curl command output?

Reply:
Linkerd may not be present (not installed, not up yet).
In that case the curl prints an error message and the user may be confused and believing that something is wrong.
The message is:

> curl: (7) Failed to connect to localhost port 4191: Connection refused

or similar.
As the situation is not truly an error the message is better suppressed.

Reply:
:+1: sure.

Reply:
Done, commit ede07bf70e394c3c45446f024f0432c131e70ada

Reply:
I remember Loic and me wondering about this curl error message. Good thing to hide it.